src/main/java/org/jenkinsci/plugins/proccleaner/PsProcess.java

```diff
@@ -116,8 +117,11 @@ public boolean equals(Object o) {
                 return true;
         return false;
     }
-    
-    //TODO implement hashCode etc.
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(pid, ppid, args, ptree);
```

Comment:
Careful the `hashCode` is out of sync with `equals`

Reply:
Good spot! I'd change ```hashCode()``` following ```equals()``` impl. although I'm not happy with the original...